copyright.txt

```diff
@@ -1,9 +1,5 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
 
```

Comment:
Should we also consider including the source for `dog_icon.png` in this file?

Reply:
Good point, will include!